projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/style.scss

```diff
@@ -37,19 +37,16 @@ $border_radius: math.div($bar-height, 2);
         font-size: var(--font-body);
         line-height: 24px;
     }
-}
-
-.components-popover {
-    width: calc(100% - 63px);
-    max-width: 340px;
-    min-height: 150px;
 
-    @media (min-width: 783px) {
-        width: 100%;
-    }
-    &__content {
-        width: 100%;
+    &__parent {
+        width: calc(100vw - 63px);
```

Comment:
I know this is copied from previous code, if you know the reason we are subtracting exactly 63 pixels it might be good to add a comment here as to why that value is chosen 😅

Reply:
Tbh, I don't know but I don't see any issue with changing it to 64px 🤷‍♂️

Reply:
Thanks for actually questioning that!